editor.js

```diff
@@ -82,7 +82,7 @@ function makeEditorShim(id) {
 
 
 function extension_editor_finish_run() {
-	if (typeof(unsafeWindow) !== 'undefined') {
+	if ((typeof(unsafeWindow) !== 'undefined') && (navigator.userAgent.indexOf('Mobile') === -1)) {
```

Comment:
Lemme check, I faintly remember them loading but I ain't sure right now.

OK, no, forget what I said. The extensions do not get loaded on the editor page.

`XKit.browser()` on the other hand is part of `xkit.js` and thus available on the editor page, @dlmarquis. We just have to add a mobile check to it